lib/compiler/Cargo.toml

```diff
@@ -20,17 +20,19 @@ hashbrown = { version = "0.9", optional = true }
 serde = { version = "1.0", features = ["derive"], optional = true }
 thiserror = "1.0"
 serde_bytes = { version = "0.11", optional = true }
-smallvec = "1.6" 
+smallvec = "1.6"
+borsh = { git = "https://github.com/near/borsh-rs", rev = "c62cdfbd10d4a17fc877809eba4ccb65e866d5f8", optional = true }
```

Comment:
We shouldn't merge this while there's a git dependency as it will block us from doing releases on crates.io

Reply:
This commit haven't published, will keep this updated as we publish a new borsh